src/style.css

```diff
@@ -1,3 +1,112 @@
+
 body {
   margin: 0;
 }
+
+.header {
+  background-color: #fff;
+  color: black;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+}
+
+.container {
+  width: 1200px;
+  height: 60px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin: 0;
+  padding: 0;
+  margin-right: 20px;
+}
+
+.nav__link {
+  margin: 0;
+  padding: 23px 0;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  position: relative;
+  padding-bottom: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
move font-family to the `html` tag selector

Reply:
please move font-family to the html tag selector and delete it here